### PR TITLE
Feat seed database cronjob

### PR DESCRIPTION
**Pull request by liewshanhong**
To run script: 
python manage.py runscript -v2 seed_database

To start cron in docker:
- go to docker CLI
- enter 'service cron start'

To check status of cron in docker:
- go to docker CLI
- enter 'service cron status'

To run cron:
- add job to cron 
'python manage.py crontab add'

### CODE REVIEW

File: skills_DS/scripts/seed_database.py
Comment (SanaBob) [inline skills_DS/scripts/seed_database.py:12]: this seems like a lot of data for a constant. I would rename this to LOCATION or CONSTANTS_LOCATION but this is also fine.

File: skills_DS/scripts/seed_database.py
Comment (SanaBob) [inline skills_DS/scripts/seed_database.py:39]: same here

**Review by SanaBob — APPROVED** [review]

**Comment by SanaBob** [comment]
Not sure if this PR is done yet, but you also need to update Readme, if there's update on running the docker

**Comment by liewshanhong** [comment]
waiting for one more review

**Review by brianzhouzc — APPROVED** [review]

**Pull request merged**
